gcp-observability/src/test/java/io/grpc/gcp/observability/ObservabilityConfigImplTest.java

```diff
@@ -139,4 +185,82 @@ public void eventTypes() throws IOException {
         ImmutableList.of(EventType.GRPC_CALL_REQUEST_HEADER, EventType.GRPC_CALL_HALF_CLOSE,
             EventType.GRPC_CALL_TRAILER));
   }
+
+  @Test
+  public void enableCloudMonitoringAndTracing() throws IOException {
+    observabilityConfig.parse(ENABLE_CLOUD_MONITORING_AND_TRACING);
+    assertFalse(observabilityConfig.isEnableCloudLogging());
+    assertTrue(observabilityConfig.isEnableCloudMonitoring());
+    assertTrue(observabilityConfig.isEnableCloudTracing());
+  }
+
+  @Test
+  public void alwaysSampler() throws IOException {
+    observabilityConfig.parse(GLOBAL_TRACING_ALWAYS_SAMPLER);
+    assertTrue(observabilityConfig.isEnableCloudTracing());
+    ObservabilityConfig.Sampler sampler = observabilityConfig.getSampler();
+    assertThat(sampler).isNotNull();
+    assertThat(sampler.getType()).isEqualTo(ObservabilityConfig.SamplerType.ALWAYS);
+  }
+
+  @Test
+  public void neverSampler() throws IOException {
+    observabilityConfig.parse(GLOBAL_TRACING_NEVER_SAMPLER);
+    assertTrue(observabilityConfig.isEnableCloudTracing());
+    ObservabilityConfig.Sampler sampler = observabilityConfig.getSampler();
+    assertThat(sampler).isNotNull();
+    assertThat(sampler.getType()).isEqualTo(ObservabilityConfig.SamplerType.NEVER);
+  }
+
+  @Test
+  public void probabilisticSampler() throws IOException {
+    observabilityConfig.parse(GLOBAL_TRACING_PROBABILISTIC_SAMPLER);
+    assertTrue(observabilityConfig.isEnableCloudTracing());
+    ObservabilityConfig.Sampler sampler = observabilityConfig.getSampler();
+    assertThat(sampler).isNotNull();
+    assertThat(sampler.getType()).isEqualTo(ObservabilityConfig.SamplerType.PROBABILISTIC);
+    assertThat(sampler.getProbability()).isEqualTo(0.75);
+  }
+
+  @Test
+  public void bothSamplerAndSamplingRate_error() throws IOException {
```

Comment:
Also add test for when neither one is set.

Reply:
The existing tests (before this PR) have that - there is neither one. All tests before `io.grpc.gcp.observability.ObservabilityConfigImplTest.alwaysSampler()` test that